java/test/behaviour/query/language/rulevalidation/RuleValidationTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.vaticle.typedb.driver.test.behaviour.query.language.rulevalidation;;
+
+import com.vaticle.typedb.driver.test.behaviour.BehaviourTest;
+import io.cucumber.junit.Cucumber;
+import io.cucumber.junit.CucumberOptions;
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(
+        strict = true,
+        plugin = "pretty",
+        glue = "com.vaticle.typedb.driver.test.behaviour",
+        features = "external/vaticle_typedb_behaviour/query/language/rule-validation.feature",
+        tags = "not @ignore and not @ignore-driver and not @ignore-typedb-driver-java"
+)
+public class RuleValidationTest extends BehaviourTest { // TODO: How to add to automation.yml?
```

Comment:
The TODO says it all. I didn't really understand how to correctly include it to automation.yml, because there are different sets of query/language tests over there... Would be glad to get a direct guidance.